docs/cookbook/entities/custom-translatable-model.rst

```diff
@@ -451,16 +443,14 @@ To have templates for your Entity administration out of the box you can use Grid
 
 Having a grid prepared we can configure routing for the entity administration:
 
-Create the ``app/config/routing/admin/supplier.yml`` file. Include it in the ``app/config/routing/admin/admin.yml``, which
-should be also included in the ``config/routes.yaml``.
-
 .. code-block:: yaml
 
-    # config/routes/admin/supplier.yaml
+    # config/routes.yaml
     app_admin_supplier:
         resource: |
             alias: app.supplier
             section: admin
+            path: admin
```

Comment:
If I understood you correctly, you want to put this routing behind the admin firewall. IMHO, a better solution would be to add `prefix: admin` after `type: sylius.resource`, so we will keep structure: `admin/suppliers/*`.